core/src/main/java/org/apache/accumulo/core/util/HostAndPort.java

```diff
@@ -143,7 +143,7 @@ public static HostAndPort fromParts(String host, int port) {
    *           if nothing meaningful could be parsed.
    */
   public static HostAndPort fromString(String hostPortString) {
-    hostPortString = java.util.Objects.requireNonNull(hostPortString);
+    java.util.Objects.requireNonNull(hostPortString);
```

Comment:
Maybe add the optional message stating that hostPortString was null?  The way that the NPE shows up in the logs, adding the message seems to help figuring things out.